services/director-v2/tests/unit/conftest.py

```diff
@@ -420,3 +425,17 @@ def mock_docker_api(mocker: MockerFixture) -> None:
         "simcore_service_director_v2.modules.dynamic_sidecar.scheduler.task.get_dynamic_sidecar_state",
         return_value=(ServiceState.PENDING, ""),
     )
+
+
+@pytest.fixture
+async def docker() -> AsyncIterable[aiodocker.Docker]:
```

Comment:
MINOR: i would rename the fixture ``docker_client`` since ``docker`` is also a package and could be confusing

Reply:
actually that would collide. I've got for `async_docker_client` and also refactored it in another test module.